src/xprs_env.jl

```diff
@@ -5,16 +5,23 @@
 Initialize Xpress environment.
 Need to do ir once and only once.
 """
-function Env()
-    ret = @xprs_ccall(init, Cint, (Ptr{Cchar},), C_NULL)
-    if ret != 0
-        if ret == :number
-            error("Invalid Xpress license")
-        else
-            error("Failed to create environment (error $ret).")
+mutable struct Env
```

Comment:
not sure we should keep this...
it does not hold info like gurobi's, its just a boot in the library